main.js

```diff
@@ -0,0 +1,102 @@
+var board = {
+  // Initialize starting board for game
+  startingPegBoard: [
+                      [5, 4, 3, 2, 1],
```

Comment:
I'm not sure what code formatter you use but I would make sure to set indentation rules in the config and adhere to them. Your editor will do this for you, either on save or manually.